WolvenKit/Views/Templates/RedCNameEditor.xaml.cs

```diff
@@ -161,6 +161,10 @@ public string Hash
             }
             set
             {
+                if (string.IsNullOrEmpty(value))
```

Comment:
This doesn't work, deleting the text in a CName editor doesn't set the Hash to 0

Reply:
I have reverted it